widget/diagramwidget/connectionpad.go

```diff
@@ -223,11 +234,9 @@ type RectanglePad struct {
 
 // NewRectanglePad creates a RectanglePad and associates it with the DiagramElement. The size of the
 // pad becomes the size of the padOwner.
-func NewRectanglePad(padOwner DiagramElement) *RectanglePad {
+func NewRectanglePad() *RectanglePad {
```

Comment:
With this change of signature the documents no longer correctly describe what it does.
What this API break intentional?